block_function.rb

```diff
@@ -0,0 +1,24 @@
+#(6) Create a program block_function.rb.  It should have a function do_calc that calls a block using a yield statement. 
```

Comment:
This is a really cool solution! It's a little more than I think was expected, but there's no problem in practicing classes and creating instances. :) You can skip the class and just make do with a `do_calc` function though. I interpreted the question a little differently, but you've got the concepts down so ✅